.github/workflows/ci-cd.yml

```diff
@@ -312,18 +311,6 @@ jobs:
                 }}",
                 file=outputs_file,
             )
-    - name: Set the expected dist name
-      id: dist
-      run: |
-        from os import environ
-        from pathlib import Path
-
-        FILE_APPEND_MODE = 'a'
-
-        with Path(environ['GITHUB_OUTPUT']).open(
-                mode=FILE_APPEND_MODE,
-        ) as outputs_file:
-            print("name=${{ env.dists-name }}", file=outputs_file)
```

Comment:
This is referenced in the job outputs around line 115/116 and is than used in the GitHub Environment URL declaration in the PyPI publishing job (line 1007/994). We need to be careful to keep that reference. Especially since that may only become evident at the time of release as that job won't run on PRs.

So instead of a blind removal, let's keep this and maybe the env var too. We could then keep reusing it for the wheel and only update that sdist bit.

Reply:
Marking as resolved.